api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -953,8 +953,15 @@ message Cluster {
   // :ref:`STRICT_DNS<envoy_v3_api_enum_value_config.cluster.v3.Cluster.DiscoveryType.STRICT_DNS>`
   // and :ref:`LOGICAL_DNS<envoy_v3_api_enum_value_config.cluster.v3.Cluster.DiscoveryType.LOGICAL_DNS>`
   // this setting is ignored.
-  google.protobuf.Duration dns_refresh_rate = 16
-      [(validate.rules).duration = {gt {nanos: 1000000}}];
+  // This field is deprecated in favor of using the :ref:`cluster_type<envoy_v3_api_field_config.cluster.v3.Cluster.cluster_type>`
+  // extension point and configuring it with :ref:`DnsCluster<envoy_v3_api_msg_extensions.clusters.dns.v3.DnsCluster>`.
+  // If :ref:`cluster_type<envoy_v3_api_field_config.cluster.v3.Cluster.cluster_type>` is configured with
+  // :ref:`DnsCluster<envoy_v3_api_msg_extensions.clusters.dns.v3.DnsCluster>`, this field will be ignored.
+  google.protobuf.Duration dns_refresh_rate = 16 [
```

Comment:
Do these fields still need to be used for other cluster types, like redis?  If so, we probably can't mark them as deprecated yet.

Reply:
Both redis and dynamic forward proxies define their own refresh rates

https://github.com/envoyproxy/envoy/blob/983545fb66f7b976d4e94fedc12324dcc5e479be/api/envoy/extensions/clusters/redis/v3/redis_cluster.proto#L58-L85

https://github.com/envoyproxy/envoy/blob/983545fb66f7b976d4e94fedc12324dcc5e479be/api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto#L65

Couldn't find any other ones.